Week6.js

```diff
@@ -0,0 +1,121 @@
+
+// Q1
+//console.log('Hello world!');
+//console.log('!مرحبا بالعالم');
+//console.log('Saluton mondo');
+//console.log('Hej världen!');
+//console.log('Selam Dünya!');
+//console.log('ہیلو ورلڈ!');
+//console.log('Aloha kākou honua!');
+
+//Q2
+//console.log('Im awesome');
```

Comment:
We should still keep the quote in the string!

You can escape the quote like this:
```javascript
console.log('I\'m awesome');
```

Or use double quotes for your outer quotes like this:
```javascript
console.log("I\'m awesome");
```